assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.stories.js

```diff
@@ -51,11 +50,7 @@ export default {
 	decorators: [
 		withQuery,
 		( Story, { args } ) => {
-			return (
-				<WithRegistrySetup func={ () => {} }>
-					<Story { ...args } />
-				</WithRegistrySetup>
-			);
+			return <Story { ...args } />;
```

Comment:
There are still a few of these spreads left behind, but this decorator is not changing the output at all and can be removed entirely.